backend/internal/models/media.go

```diff
@@ -17,8 +17,14 @@ type Media interface {
 	GetMediaType() MediaType
 }
 
+// names dont get much better than this
+type MediaWithReviewCountAndType struct {
+	Media Media     `json:"media"`
+	Count int       `json:"review_count"`
+	Type  MediaType `json:"type"`
+}
+
 type Album struct {
-	Media       MediaType `json:"media"`
```

Comment:
just curious, what was the reason for taking this out? especially since the field ends up being added to MediaWithReviewCountAndType, do you think it's worth keeping it here so that any other media-related endpoint can use it too? (we should prob rename it to MediaType tho)

(mainly asking because the sort=recent version of this endpoint now no longer has an obvious type flag since it's just returning `[]models.Media`)

Reply:
I took it out because I didn't think album struct needed to have mediatype field and be like self aware that it's an album. Same with track. But I can add it back

Reply:
ah yeah i get that, them being self aware feels more logical but i think for the sake of the API responses (and so the frontend can easily tell them apart) we should keep it